notebooks/code.py

```diff
@@ -0,0 +1,27 @@
+import dataiku
+
+def list_notebooks(client=None):
```

Comment:
To be more flexible, `list_notebooks()` should also take a `project_key` argument and return results for a single project. This way, you can call it in a loop if you want to iterate over multiple projects. You can also pass the client as argument in order to instantiate it only once.

Reply:
If the goal is to _print_ the list of notebooks and not _list_ it (i.e. return it in the form of a list or a dict), then the name of the function should clearly reflect it. At the minimum, you should make it clear in the function's docstring.